server/database/models/donorHomepageModel.ts

```diff
@@ -0,0 +1,104 @@
+import { Document, model, Schema, Types } from "mongoose";
+
+interface MapPoint {
+    x: number;
+    y: number;
+}
+
+interface MapTestimonial {
+    id: number;
+    imagePath: string;
+    testimonial: string;
+}
+
+interface Statistic {
+    icon: string;
+    measurement: string;
+    stat: string;
+}
+
+interface ClientTestimonial {
+    image: string;
```

Comment:
consider changing to `imagePath` to keep it consistent with the `MapTestimonial` interface, or rename the other to `image`

also might be useful to add an `id` field like in `MapTestimonial` (to uniquely identify different client testimonials)